_lifeofinternatpuppet/post_7_and_8.md

```diff
@@ -0,0 +1,197 @@
+
+A link to the Puppet's PowerPoint Presentation was also sent by Daniel and David Swan but it wasn't used for creating a presentation on Litmus demo as the demo itself was quite informal, as said by Norman previously.
+
+Now you must be thinking how did I found the litmus demo myself, well the answer is that I was fully prepared for it, did lots and lots of practice, I was asked lots of questions by Norman during the demo session and I learnt some new commands during the session like script command, history command etc...
+
+There be more information hopefully shared by Norman on the script command in the coming weeks, so keep your eye out for useful links etc...
+
+On Wednesday 14th October 2020, I continued to work on the to do list given by Daina on Tuesday within the slack channel, I really enjoyed doing as it helped me debug errors by myself so thank you to Daiana for always helping me out when needed!
+
+Later on that morning, I also attended the EMEA Food/Chat Session where the following topics were covered by the team :-
+
+- Git bisect and conquer
+- Luchi - How to SSH into Windows?
+
+After the EMEA Food and Chat Session, I attended a session with Daiana where I was told to do a review/merge once the rubucop bug failure was fixed. This was done successfully.
+I attended the Show and Tell Session from 2pm to 3:30pm.
+
+I also informed Norman about my GP appointment on Friday and shared my plans for the QUB Graduate Recruitment and Placement Fair which will take place on Thursday 22nd October 2020 from 2pm-6pm.
+I am really looking forward to the event!
+
+I emailed QUB Careers for a 1:1 appointment for next week, regarding my September monthly log approved by Norman earlier in the week.
+
+On Thursday 15th October 2020 I started off my morning by setting up some 1:1 meetings on Lattice and scheduled training sessions with Michael on Windows Powershell and DSC Builder for next week.
+
+After doing that, I attended a short demo on releasing the puppetlabs-firewall module, where Daiana showed me the process once again. Daiana did the release stage of the process while she gave me the responsibility of doing the tag process for that module and raise a pr for review before mergeback took place in the final stage.
+
+Once this was done, Daiana and I attended a session delivered by David Swan on Networking Modules. I really enjoyed that knowledge sharing session and plan to get some hands-on experience when I am free as I would love to explore what it will include, just out of curiosity!
+
+I shared the training and show and tell session recordings on the shared Google Drive Space with the IAC Team.
+
+During the second part of releasing a module with Daiana, I become familiar with some new git commands like git branch -a, git diff and git log to name a few!
```

Comment:
Just a minor suggestion. When referencing commands, it's nice to wrap them in formatting:
```suggestion
During the second part of releasing a module with Daiana, I become familiar with some new git commands like `git branch -a`, `git diff` and `git log` to name a few!
```